third_party/bigframes_vendored/geopandas/geoseries.py

```diff
@@ -132,3 +132,83 @@ def from_xy(cls, x, y, index=None, **kwargs) -> bigframes.geopandas.GeoSeries:
                 A GeoSeries of Point geometries.
         """
         raise NotImplementedError(constants.ABSTRACT_METHOD_ERROR_MESSAGE)
+
+    @classmethod
+    def from_wkt(
+        cls, data, index=None, crs=None, **kwargs
+    ) -> bigframes.geopandas.GeoSeries:
+        """
+        Alternate constructor to create a GeoSeries from a list or array of
+        WKT objects.
+
+        **Examples:**
+
+            >>> import bigframes.geopandas
+            >>> bpd.options.display.progress_bar = None
+
+            >>> wkts = [
+            ... 'POINT (1 1)',
+            ... 'POINT (2 2)',
+            ... 'POINT (3 3)',
+            ... ]
+            >>> s = bigframes.geopandas.GeoSeries.from_wkt(wkts)
+            >>> s
+            0    POINT (1 1)
+            1    POINT (2 2)
+            2    POINT (3 3)
+            dtype: geometry
+
+        Args:
+            data (array-like):
+                Series, list, or array of WKT objects.
+
+            index (array-like or Index, optional):
+                The index for the GeoSeries.
+
+            crs (optional):
+                Coordinate Reference System of the geometry objects.
+                Coordinate Reference System of the geometry objects. Can be
+                anything accepted by pyproj.CRS.from_user_input(), such as an
+                authority string (eg “EPSG:4326”) or a WKT string.
+
+            **kwargs:
+                Additional arguments passed to the Series constructor, e.g. `name`.
+
+        Returns:
+            bigframes.geopandas.GeoSeries:
+                A GeoSeries of geometries.
+        """
+        raise NotImplementedError(constants.ABSTRACT_METHOD_ERROR_MESSAGE)
+
+    def to_wkt(self, **kwargs) -> bigframes.series.Series:
```

Comment:
Since we don't support the shapely args (https://shapely.readthedocs.io/en/2.0.6/reference/shapely.to_wkt.html or https://numpy.org/doc/stable/reference/ufuncs.html#ufuncs-kwargs), let's delete `**kwargs`. This should fix `mypy`, too.